build.gradle.kts

```diff
@@ -17,41 +15,47 @@ repositories {
     mavenCentral()
 }
 
-dependencies {
-    api("org.jetbrains.kotlinx:kotlinx-serialization-json:1.4.1")
-    testImplementation(kotlin("stdlib"))
-    testImplementation(kotlin("test"))
-    testImplementation("org.junit.jupiter:junit-jupiter-api:5.9.0")
-    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.9.0")
-}
-
-java {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
-    withJavadocJar()
-    withSourcesJar()
```

Comment:
sources jar now contains test code, why?

Reply:
tested locally, it doesn't contain test code:
![image](https://user-images.githubusercontent.com/1164369/201644226-12755622-d0ee-4ff2-b007-f9431edc617b.png)

Reply:
ohh, I see it does exist in `io/github/detekt/sarif4k/sarif4k/0.3.0-SNAPSHOT/sarif4k-0.3.0-SNAPSHOT-sources.jar` (kotlinMultiplatform artifact).
Will check it

Reply:
it's even questionable if the multiplatform artifact needs sources at all...

I checked coroutines-core and core-jvm artifacts, and they have the same, I think it's safe to go with this. I just have never seen test sources published before KMP.